cmd/minikube/cmd/config/validations.go

```diff
@@ -125,3 +127,19 @@ func IsValidAddon(name string, val string) error {
 	}
 	return errors.Errorf("Cannot enable/disable invalid addon %s", name)
 }
+
+func IsContainerdRuntime(_, _ string) error {
+	config, err := config.Load()
+	if err != nil {
+		return fmt.Errorf("error getting cluster config: %v", err)
+	}
+	if config.KubernetesConfig.ContainerRuntime != constants.ContainerdRuntime {
+		return fmt.Errorf(`This addon can only be enabled with the containerd runtime backend.
+
+To enable this backend, please start minikube again with the following flags:
```

Comment:
can you please clarify if user has to "delete" and "start" or just "stop" and "start"?

Reply:
done